sql/xsql-shell/src/main/scala/org/apache/spark/sql/xsql/shell/SparkXSQLShell.scala

```diff
@@ -161,7 +161,7 @@ object SparkXSQLShell extends Logging {
     def run(commands: String, silent: Boolean = false) = {
       var startTime = System.currentTimeMillis()
       for (command <- commands.split(";")) {
-        var sql = command.trim()
+        var sql = command.trim().replaceAll("[\\t\\n\\r]", " ")
```

Comment:
If SQL contains '\t' or '\n', what results will be?

Reply:
Sorry, this PR I haven't considered this way clearly.